src/pages/registration/RegistrationForm.tsx

```diff
@@ -83,6 +84,7 @@ const RegistrationForm = ({ identifier, isNewRegistration }: RegistrationFormPro
     <Forbidden />
   ) : (
     <StyledRegistration>
+      <SkipLink href="#form">{t('common:skip_to_schema')}</SkipLink>
```

Comment:
For at det skal virke 100% etter å ha "hoppet til hovedinnhold" på Firefox må denne flyttes til _under_ `RegistrationPageHeader`, men da må man tabbe gjennom "Tilbake" og evt "Vis hele tittelen" på Chrome og Edge, så tror vi bare legger FF litt til siden her. Vet ikke hvorfor, men på FF må man hoppe bakover med SHIFT+TAB for at den skal få fokus 🤷‍♂️ 
Fint om du tester med safari @sarahse .
1. Åpne en registrering
2. TAB -> hopp til hovedinnhold -> ENTER
3. TAB -> hopp til skjema -> ENTER
4. Første inputfelt skal ha fokus

Reply:
Den "Hopp til hovedinnhold"-greia vises ikke i det hele tatt i safari. Kommer rett til skjemaet ved første tab...

https://user-images.githubusercontent.com/6595982/114680494-8d8e9700-9d0d-11eb-8bd9-691db3b48fbd.mov

Reply:
tab-navigering fungerer ikke i det hele tatt på safari :) :) :) :) :)

Reply:
never mind - det fungerer. men en må tydeligvis bruke option+tab for å tabbe 😅

Reply:
https://user-images.githubusercontent.com/6595982/114681236-3dfc9b00-9d0e-11eb-8fae-eddf06dd9260.mov

Reply:
Får ikke til å se klippene... men er det vanlig at du må bruke option+tab for å tabbe?
det skal virke likt hos oss som på uutilsynet.no

Reply:
hm, rart. men nei, det er tydeligvis sånn på safari. https://www.scottohara.me/blog/2014/10/03/link-tabbing-firefox-osx.html